unity-ktwo/Assets/Scripts/Modifiers/Damagable.cs

```diff
@@ -20,6 +20,12 @@ protected void Start()
         }
     }
 
+    virtual public void Heal(float healAmount)
+    {
+        currentHealth += healAmount;
```

Comment:
repost:

optional suggestion: 
happened to rethink this as a one liner:
```c#
currentHealth = Mathf.Min(currentHealth + healAmount, startingHealth);
```

Reply:
changed